kahuna/public/stylesheets/main.css

```diff
@@ -282,7 +282,7 @@ a:focus {
     width: 100%;
 }
 
-.right {
+.flex-right {
```

Comment:
meep, this shouldn't really be part of this PR. `flex-right` felt like a more appropriate name for this class.